osu.Game.Tests/NonVisual/CustomDataDirectoryTest.cs

```diff
@@ -151,16 +152,19 @@ public void TestMigration()
                     Assert.That(!host.Storage.ExistsDirectory(Path.Combine("test-nested", "cache")));
                     Assert.That(storage.ExistsDirectory(Path.Combine("test-nested", "cache")));
 
-                    foreach (var file in OsuStorage.IGNORE_FILES)
+                    if (osuStorage != null)
```

Comment:
Shouldn't this be an actual assertion, as in the following?

```csharp
Assert.That(osuStorage, Is.Not.Null);
```

Imagine this regresses for some reason and `osuStorage` becomes null; this `if` will then mean that the test will silently skip over the nested asserts and stop testing if the ignore lists work at all.

(As an aside, I think it'd also make sense to change type of `osuStorage` to `MigratableStorage` as it's the most general type possible in this scenario)

Reply:
Good point! I'll add that and change the cast right away. I'll also include the property accessor change right away.